website/views.py

```diff
@@ -37,4 +41,50 @@ def register(request):
             return render(request, 'register.html', {'form': form})
     else:
         form = SignUpForm()
-        return render(request, 'register.html', {'form': form})
\ No newline at end of file
+        return render(request, 'register.html', {'form': form})
+    return render(request, 'register.html', {'form': form})
+
+
+def record(request,pk):
```

Comment:
Use class based views and mention authentication class to avoid `request.user.is_authenticated`